web/cypress/fixtures/hfi-calc/dailies-disable-station.json

```diff
@@ -1111,7 +1111,7 @@
                 "rate_of_spread": 0.001902355415765236,
                 "hfi": 1.8253517760238567,
                 "intensity_group": 1,
-                "sixty_minute_fire_size": 9.998822417249202e-07,
+                "sixty_minute_fire_size": 9.998822417249202e-7,
```

Comment:
another prettier thing. seems weird. is it ok?

Reply:
These are test fixtures, so as long as tests pass it's fine. Looks like just removing leading zeroes from floating point exponent notation.